.github/workflows/ci.yml

```diff
@@ -67,6 +67,40 @@ jobs:
       - name: Run integration tests
         run: cargo test --release --locked --package integration-tests
 
+  api-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: "./.github/actions/init"
+
+      - name: Run tests
+        working-directory: pop-api
+        run: cargo test --release --all-features
+
+  api-integration-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: "./.github/actions/init"
+
+      - name: Run integration tests
+        working-directory: pop-api/integration-tests
+        run: cargo test --release
+
+      - name: Copy contract WASM files to artifacts folder
```

Comment:
Not sure if I see the point in this. Is it just so that we can download them and put them in the right place on dev machines to avoid a build? Seems a hassle if so and probably something that will go unused, especially as the integration tests build script will sort that automatically. I see more value in always have the latest contract wasm on a dev machine so integration tests work accordingly.

Reply:
this is valid. I take back what I said about storing the artifacts.